crates/e2e-test-utils/src/testsuite/mod.rs

```diff
@@ -0,0 +1,100 @@
+//! Utilities for running e2e tests against a node or a network of nodes.
+
+use actions::{Action, ActionBox};
+use eyre::Result;
+use setup::Setup;
+use std::{future::Future, pin::Pin};
+
+pub mod actions;
+pub mod setup;
+
+#[cfg(test)]
+mod examples;
+
+/// A boxed future type for async trait methods
+pub type BoxFuture<'a, T> = Pin<Box<dyn Future<Output = T> + Send + 'a>>;
```

Comment:
I believe this is the same as `futures::BoxFuture`

Reply:
awesome thx!